bin/alluxio-start.sh

```diff
@@ -299,6 +311,7 @@ main() {
         stop all
         sleep 1
       fi
+      start_logserver
```

Comment:
only start if the log server is enabled, same below

Reply:
Done.